src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -10,12 +10,6 @@ export function focusElement(selectorOrElement, options) {
   const el = $(selectorOrElement);
 
   if (el) {
-    if (el.tabIndex === 0) {
-      el.setAttribute('tabindex', '0');
-    }
-    if (el.tabIndex < 0) {
-      el.setAttribute('tabindex', '-1');
-    }
     el.focus(options);
```

Comment:
Could we simplify this with optional chaining?

```suggestion
  el?.focus(options);
```

Reply:
Yeah, but I'm old, and ES2020 is too young and cool for me

Reply:
We don't have an ESLint rule for it, but optional chaining should be preferred when possible.